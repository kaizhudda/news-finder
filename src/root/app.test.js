import React from 'react';
import { shallow } from 'enzyme';
import App from './app.ui';

let wrapped = shallow(<App />);

describe('App', () => {
  it('should render the App Component', () => {
    expect(wrapped).toMatchSnapshot();
  });

  it('should render Header', () => {
    expect(wrapped.find('Connect(Header)').exists()).toBe(true);
  });

  it('should render SubHeader', () => {
    expect(wrapped.find('Connect(SubHeader)').exists()).toBe(true);
  });

  it('should render Routes', () => {
    expect(wrapped.find('Routes').exists()).toBe(true);
  });

});