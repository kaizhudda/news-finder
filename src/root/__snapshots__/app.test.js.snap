// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the App Component 1`] = `
<Route>
  <Component />
</Route>
`;
